### PR TITLE
Also accept "WARN" and "ERR" as valid log level names

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

File: logging/levels.go [hunk 1]
```diff
@@ -77,9 +77,9 @@ func LookupLevel(name string) (slog.Level, error) {
 		return LevelInfo, nil
 	case levelNoticeName:
 		return LevelNotice, nil
-	case levelWarningName:
+	case levelWarningName, "WARN":
```
Comment (drevell) [inline logging/levels.go:80]: Should the same pattern be applied to the other levels? Is there a reason for these two to behave differently?

**Review by drevell — CHANGES_REQUESTED** [review]

File: logging/levels.go (hunk 1 above)
Comment (sethvargo) [reply logging/levels.go:80]: This is to maintain backwards compat with the old loggers levels. The other levels remain the same. I can document that if desired.

Comment (drevell) [reply logging/levels.go:80]: Yeah, might be worth a note. I don't have strong feelings on it.

**Review by drevell — APPROVED** [review]

File: logging/levels.go (hunk 1 above)
Comment (sethvargo) [reply logging/levels.go]: Added docs.

**Review by drevell — APPROVED** [review]

**Pull request merged**